.github/workflows/unit-tests.yml

```diff
@@ -109,52 +149,48 @@ jobs:
         run: |
           brew install cmake boost sdl2 sdl2_mixer gettext miniupnpc libiconv
           echo /usr/local/opt/gettext/bin >> $GITHUB_PATH
+          ls -la /usr/local/Cellar/boost/
           BOOST_ROOT=$(find /usr/local/Cellar/boost/${BOOST_VERSION}_* -maxdepth 0)
           echo "BOOST_ROOT=${BOOST_ROOT}" >> $GITHUB_ENV
 
       - name: Setup CCache
         uses: hendrikmuhs/ccache-action@v1
         with:
-          key: ${{matrix.os}}-${{matrix.compiler}}-${{matrix.buildType}}-${{matrix.boostVersion}}
+          key: ${{matrix.os}}-${{matrix.compiler}}-${{matrix.type}}-${{matrix.boost}}
           max-size: 200M
 
-      - name: Install dependencies
+      - name: Compile CMake (Linux)
         if: "!startsWith(runner.os, 'macos')"
         run: |
-          # Minimum requirements: CMake 3.9, Boost 1.69
           CMAKE_DIR="${DEPS_DIR}/cmake"
           external/libutil/tools/ci/installCMake.sh "${CMAKE_VERSION}" "${CMAKE_DIR}"
           echo "${CMAKE_DIR}/bin" >> $GITHUB_PATH
 
+      - name: Compile Boost (Linux)
+        if: "!startsWith(runner.os, 'macos')"
+        run: |
           BOOST_ROOT="${DEPS_DIR}/boost${BOOST_VERSION}"
           echo "BOOST_ROOT=${BOOST_ROOT}" >> $GITHUB_ENV
           echo "ADDITIONAL_CMAKE_FLAGS=${ADDITIONAL_CMAKE_FLAGS} -DBoost_NO_SYSTEM_PATHS=ON -DBoost_NO_BOOST_CMAKE=ON" >> $GITHUB_ENV
           external/libutil/tools/ci/installBoost.sh "${BOOST_VERSION}" "${BOOST_ROOT}" "filesystem,system,program_options,thread,test,locale,iostreams,regex" shared
           cat /tmp/boost.log || true
           ls -la ${BOOST_ROOT}/lib/ || true
 
-      - name: Enable coverage collection
-        if: matrix.coverage
-        run: echo "ADDITIONAL_CMAKE_FLAGS=${ADDITIONAL_CMAKE_FLAGS} -DRTTR_ENABLE_COVERAGE=ON" >> $GITHUB_ENV
-
-      - name: Set C++ standard
-        if: matrix.cxxStandard
-        run: echo "ADDITIONAL_CMAKE_FLAGS=${ADDITIONAL_CMAKE_FLAGS} -DCMAKE_CXX_STANDARD=${{matrix.cxxStandard}}" >> $GITHUB_ENV
-
       - name: Setup environment
-        run: |
-          echo "BUILD_TYPE=${{matrix.buildType}}" >> $GITHUB_ENV
-          echo "TRAVIS_BUILD_DIR=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+        run: echo "TRAVIS_BUILD_DIR=$GITHUB_WORKSPACE" >> $GITHUB_ENV
```

Comment:
As we'll never use travis again, maybe rather change that in the script(s) and get rid of this step which is now only for `$TRAVIS_BUILD_DIR` and rename `tools/ci/travisBuild.sh` to `githubBuild.sh` or just `build.sh`?

This is an ugly clutch anyway the build script should have been able to handle either variable.

Reply:
I don't know if the script is used at another place, so I'll leave it like it is for now